src/sss_client/common.c

```diff
@@ -767,6 +767,15 @@ static enum sss_status sss_cli_check_socket(int *errnop,
         myself_ino = myself_sb.st_ino;
     }
 
+    /* check if the socket has beed was hijacked */
```

Comment:
... has been was ... 

No need to fix this if this is the only change you have to do.

Reply:
Fixed.